blog/mapping-consensus.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Mapping consensus
+slug: mapping-consensus
+authors: abailly-iohk dnadales
+tags: [consensus]
+hide_table_of_contents: false
+---
+
+## On the need for improving consensus documentation  
+
+<!-- Where were we and what were the problems -->
+
+Consensus was developed trying to meet strict deadlines: first IOG had to
+release the Byron rewrite as soon as possible, and later on, the release of
+Shelley became the next urgent thing. In spite of working under such high
+pressure, the consensus team did its best not to compromise on assurance,
+architecture, and documentation. A proof of the latter, is the [consensus
```

Comment:
> A proof of the latter, ...

Hmm. This sentence seems a bit inaccurate to me; Thomas and Edsko wrote that as they were leaving. The document didn't exist until after the Shelley hard fork happened. Thus this paragraph is currently not totally internally consistent.